README.adoc

```diff
@@ -136,27 +138,29 @@ configuration elements. Use these credentials to view the metrics endpoint.
 
 === Adding the annotations
 
-Create the `src/main/java/io/openliberty/guides/inventory/InventoryManager.java` class:
+Create the `InventoryManager` class in the 
+`src/main/java/io/openliberty/guides/inventory/InventoryManager.java` file::
 
 [source, java, indent=0]
 ----
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[tags=InventoryManager]
 ----
 
-You added the following metrics annotations:
+Notice these metrics annotations:
```

Comment:
Probably change to: "Notice the metrics annotations added to this class:" since @sishida suggested not to use the word "these".

Reply:
Fixed